crypto/mem_sec.c

```diff
@@ -31,6 +31,10 @@
 # include <unistd.h>
 # include <sys/types.h>
 # include <sys/mman.h>
+# if defined(OPENSSL_SYS_LINUX)
```

Comment:
Are we moving away from OPENSSL_SYS_LINUX as defined in e_os.h? It looks for 

    defined(linux) || defined(__linux__)

I see inconsistent uses of the above with __linux; OPENSSL_SYS_LINUX seems to be more common (barely). I seem to recall a prior issue with the proper symbol to use with Windows?